gslib/exception.py

```diff
@@ -31,8 +31,10 @@
 
 import six
 
-NO_URLS_MATCHED_GENERIC = 'No URLs matched'
-NO_URLS_MATCHED_TARGET = 'No URLs matched: %s'
+NO_URLS_MATCHED_PREFIX = 'No URLs matched'
+NO_URLS_MATCHED_GENERIC = (NO_URLS_MATCHED_PREFIX +
+                           '. Do the files you\'re operating on exist?')
```

Comment:
Accidentally merged before they could respond.
Will make another PR if the text doesn't work.

Reply:
I think that should be fine. There's a devilishly nuanced discussion to be had around "file" vs. "object", but I think most users understand the terms to be, generally, synonymous. If you wanted, you could change the wording from "files" to "files/objects" so that people don't accidentally try to cue off of "files" to think the issue must be within their local directory?